deps.bzl

```diff
@@ -0,0 +1,138 @@
+
+# on changes to crate depdencies, run the following command:
+# CARGO_BAZEL_REPIN=1 bazel sync --only=salus-index --only=salus-rwlock \
+# --only=salus-once --only=salus-nodefault --only=salus-default -only=sbi-index
+
+def salus_dependencies():
+    crates_repository(
+        name = "salus-index",
+        isolated = False,
+        cargo_lockfile = "@salus//bazel-locks:Cargo.Bazel.lock",
+        lockfile = "@salus//bazel-locks:cargo-bazel-lock.json",
+        packages = {
+            "arrayvec": crate.spec(
+                version = "0.7.2",
+                default_features = False,
+            ),
+            "const-field-offset": crate.spec(
+                version = "0.1.2",
+            ),
+            "const-oid": crate.spec(
+                version = "0.9.0",
+                features = ["db"],
+            ),
+            "der": crate.spec(
+                version = "0.6.0",
+                features = ["derive", "flagset", "oid"],
+            ),
+            "digest": crate.spec(
+                version = "0.10.3",
+                default_features = False,
+            ),
+            "ed25519": crate.spec(
+                version = "1.5.2",
+                default_features = False,
+                features = ["pkcs8"],
+            ),
+            "ed25519-dalek": crate.spec(
+                version = "1.0.1",
+                default_features = False,
+                features = ["u64_backend"],
+            ),
+            "generic-array": crate.spec(version = "0.14.5"),
+            "hex": crate.spec(
+                version = "0.4.3",
+                default_features = False,
+            ),
+            "hex-literal": crate.spec(version = "0.3.4"),
+            "hkdf": crate.spec(version = "0.12.3"),
+            "hmac": crate.spec(version = "0.12.1"),
+            "spki": crate.spec(version = "0.6.0"),
+            "typenum": crate.spec(version = "1.15.0"),
+            "enum_dispatch": crate.spec(
+                version = "0.3.8",
+            ),
+            "flagset": crate.spec(
+                version = "0.4.3",
+            ),
+            "memoffset": crate.spec(
+                version = ">=0.6.5",
+                features = ["unstable_const"],
+            ),
+            "riscv-decode": crate.spec(
+                version = "0.2",
+            ),
+            "static_assertions": crate.spec(
+                version = "1.1",
+            ),
+            "tock-registers": crate.spec(
+                version = "0.7",
+            ),
+            "fdt-rs": crate.spec(
+                git = "https://github.com/rivosinc/fdt-rs",
+                default_features = False,
+                rev = "4aa8151",
+            ),
+            "sha2": crate.spec(version = "0.10", default_features = False),
+            "seq-macro": crate.spec(version = "0.3.2"),
+            "zeroize": crate.spec(version = "1.5.7"),
+        },
+    )
+
+    #
+    # We need 4 variations on the spin module
```

Comment:
Should we just fix our deps on `spin` so that we're not pulling in the same crate with 4 different sets of features?

Reply:
We could, but I suggest leaving it in so that we have an example of how to deal with modules that depend on multiple versions of the same crate.

Reply:
I don't think that's a good reason to keep around these unnecessary dependencies.

Reply:
fixed